kanidmd/src/lib/constants/schema.rs

```diff
@@ -484,6 +484,154 @@ pub const JSON_SCHEMA_ATTR_ACCOUNT_VALID_FROM: &str = r#"{
     }
 }"#;
 
+pub const JSON_SCHEMA_ATTR_OAUTH2_RS_NAME: &str = r#"{
+    "attrs": {
+      "class": [
+        "object",
+        "system",
+        "attributetype"
+      ],
+      "description": [
+        "The unique name of an external Oauth2 resource"
+      ],
+      "index": [
+        "EQUALITY"
+      ],
+      "unique": [
+        "true"
+      ],
+      "multivalue": [
+        "false"
+      ],
+      "attributename": [
+        "oauth2_rs_name"
+      ],
+      "syntax": [
+        "UTF8STRING_INAME"
+      ],
+      "uuid": [
+        "00000000-0000-0000-0000-ffff00000080"
+      ]
+    }
+}"#;
+
+// ⚠️  Should this have Syntax URL instead?
```

Comment:
⚠️⚠️⚠️

Reply:
Should we have a syntax url instead? IE we validate that it really is a URL when we add the data?

Reply:
URL validation's in that email validation crate isn't it? If it's relatively trivial then ... sure? :)

Reply:
Done, added url validation.